traffic_ops/app/conf/cdn.conf

```diff
@@ -1,7 +1,7 @@
 {
     "hypnotoad" : {
         "listen" : [
-            "https://[::]:60443?cert=/etc/pki/tls/certs/localhost.crt&key=/etc/pki/tls/private/localhost.key&verify=0x00&ciphers=AES128-GCM-SHA256:HIGH:!RC4:!MD5:!aNULL:!EDH:!ED"
+            "https://[::]:60443?cert=/home/bwilli415/src/localhost.crt&key=/home/bwilli415/src/localhost.key&verify=0x00&ciphers=AES128-GCM-SHA256:HIGH:!RC4:!MD5:!aNULL:!EDH:!ED"
```

Comment:
I think you may have accidentally committed these files

Reply:
whoops. Must've popped that for running tests - should look for others like that

Reply:
Okay, I think I fixed everything (and removed it from the change history too)